File: app/src/main/java/com/seungjun/randomox/network/data/RankInfo.java
package com.seungjun.randomox.network.data;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class RankInfo extends HeaderInfo {

    @SerializedName("ranks")
    public ArrayList<RankData> ranks = new ArrayList<RankData>();

    public class RankData {


        @SerializedName("user_nick")
        public String user_nick = "";

        @SerializedName("user_point")
        public int user_point = 0;

        @SerializedName("rank")
        public int rank = 0;

    }

}

File: app/src/main/java/com/seungjun/randomox/network/data/RankInfo.kt
package com.seungjun.randomox.network.data

import com.google.gson.annotations.SerializedName

import java.util.ArrayList

data class RankInfo(@SerializedName("ranks") var ranks: ArrayList<RankData> = arrayListOf()) : HeaderInfo() {

    inner class RankData {

        @SerializedName("user_nick")
        var user_nick = ""

        @SerializedName("user_point")
        var user_point = 0

        @SerializedName("rank")
        var rank = 0

    }

}
